plover/steno_dictionary.py

```diff
@@ -245,6 +251,15 @@ def casereverse_lookup(self, value):
             if key:
                 return key
 
+    def regex_reverse_lookup(self, pattern, match_limit=10):
```

Comment:
That one I'd like to move into a config value, something like "max_search_results". I'm working on a computationally cheaper partial word lookup as well; it could work for both limits.